packages/sanity/src/_internal/cli/commands/index.ts

```diff
@@ -56,6 +58,8 @@ const commands: (CliCommandDefinition | CliCommandGroupDefinition)[] = [
   deleteDatasetCommand,
   copyDatasetCommand,
   aliasDatasetCommand,
+  enableDatasetBackupCommand,
+  disableDatasetBackupCommand,
```

Comment:
I don't think this will make a practical difference, but it will bundle the enable/disable commands a little better than they are now. Happy to make the change.